### PR TITLE
Updating Polish translation of the Settings dialog

### PR DESCRIPTION
**Pull request by oleg68**
I've updated the polish translation according the changes of the Settings Dialog #934

@Romek1234 @pawel-szajna please, check my translation

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -556,6 +571,15 @@ msgstr "Grupa audio %d"
 msgid "Audio group name"
 msgstr "Nazwa grupy audio"
 
+msgid "Audio recording:"
+msgstr "Nagrywanie audio"
+
+msgid "Auto add new devices"
+msgstr "Auto dodawać nowe urządzenia"
```
Comment (pawel-szajna) [inline po/pl.po:578]: Although this one works, I'd suggest changing "dodawać" to either "dodawanie" (in line with "zarządzanie" below) or "dodawaj" (imperative).

File: po/pl.po [hunk 2]
```diff
@@ -800,7 +830,7 @@ msgid "Device: %s (%d ms)"
 msgstr "Urzadzenie: %s (%d ms)"
 
 msgid "Direct Sound:"
-msgstr "Direct Sound:"
+msgstr "Bezpośredni dźwięk:"
```
Comment (pawel-szajna) [inline po/pl.po:833]: Should this one be translated at all? Isn't this a proper noun for Microsoft's sound engine?

File: po/pl.po [hunk 3]
```diff
@@ -671,6 +695,9 @@ msgstr "Kanal:"
 msgid "Chaumont"
 msgstr "Chaumont"
 
+msgid "Check on startup"
+msgstr "Sprawdzać przy starcie"
```
Comment (pawel-szajna) [inline po/pl.po:699]: As with the previous comment, "sprawdzanie" or "sprawdzaj" could sound a little bit more natural. It feels a bit strange (at least for me) when infinitive forms (such as "sprawdzać") are used in user interface and, for some reason, they seem to be very frequently used by various malicious/scamming websites which may be one of the reasons why they sound wrong in this context.

File: po/pl.po [hunk 4]
```diff
@@ -1264,6 +1297,9 @@ msgstr "Wczytuję..."
 msgid "Log messages"
 msgstr "Log komunikatów"
 
+msgid "Loop loading:"
+msgstr "Ładowanie pętle"
```
Comment (pawel-szajna) [inline po/pl.po:1301]: "pętli" instead of "pętle"

File: po/pl.po (hunk 2 above)
Comment (oleg68) [reply po/pl.po:833]: No, it isn't. It is a lable on the "Reverb tab" that means whether the original sound should be mixed to the reverb.

File: po/pl.po [hunk 5]
```diff
@@ -1374,9 +1416,18 @@ msgstr "Mersenne Spinett 2"
 msgid "Mersenne's Improved Meantone 1"
 msgstr "Mersenne's getestet mitteltönig 1"
 
+msgid "Perform strict ODF"
+msgstr "Przeprowadzić ścisłą kontrolę ODF"
```
Comment (pawel-szajna) [inline po/pl.po:1420]: Once again, I'd suggest "przeprowadzaj ścisłą kontrolę"/"przeprowadzanie ścisłej kontroli"

File: po/pl.po [hunk 6]
```diff
@@ -1455,6 +1506,9 @@ msgstr "Północ"
 msgid "Number %s contains locale dependent floating point"
 msgstr "Liczba %s zawiera lokalnie zależną liczbę zmiennoprzecinkową"
 
+msgid "Number of CPU cores:"
+msgstr "Ilość rdzeni CPU:"
```
Comment (pawel-szajna) [inline po/pl.po:1510]: "Liczba", as "ilość" is for uncountable nouns while "liczba" is for countable

File: po/pl.po [hunk 7]
```diff
@@ -1480,6 +1534,12 @@ msgstr "Organy"
 msgid "Organ &Properties"
 msgstr "&Właściwości organów."
 
+msgid "Organ packages"
+msgstr "Pakiety organowy"
```
Comment (pawel-szajna) [inline po/pl.po:1538]: organowy -> organowe

File: po/pl.po (hunk 2 above)
Comment (pawel-szajna) [reply po/pl.po:833]: Oh, if that's the context then it's valid by all means.

File: po/pl.po (hunk 4 above)
Comment (oleg68) [reply po/pl.po:1301]: OK. Could you suggest changes with the GitHub interface that I could apply them easier?

File: po/pl.po (hunk 1 above)
Comment (pawel-szajna) [reply po/pl.po:578]:
```suggestion
msgstr "Auto dodawanie nowych urządzeń"
```

File: po/pl.po (hunk 3 above)
Comment (pawel-szajna) [reply po/pl.po:699]:
```suggestion
msgstr "Sprawdzanie przy starcie"
```

File: po/pl.po (hunk 4 above)
Comment (pawel-szajna) [reply po/pl.po:1301]:
```suggestion
msgstr "Ładowanie pętli"
```

File: po/pl.po (hunk 5 above)
Comment (pawel-szajna) [reply po/pl.po:1420]:
```suggestion
msgstr "Przeprowadzaj ścisłą kontrolę ODF"
```

File: po/pl.po (hunk 6 above)
Comment (pawel-szajna) [reply po/pl.po:1510]:
```suggestion
msgstr "Liczba rdzeni CPU:"
```

File: po/pl.po (hunk 7 above)
Comment (pawel-szajna) [reply po/pl.po:1538]:
```suggestion
msgstr "Pakiety organowe"
```

File: po/pl.po [hunk 8]
```diff
@@ -1480,6 +1534,12 @@ msgstr "Organy"
...
+
+msgid "Organ packages:"
+msgstr "Pakiety organowy:"
```
Comment (pawel-szajna) [inline po/pl.po:1541]:
```suggestion
msgstr "Pakiety organowe:"
```

File: po/pl.po [hunk 9]
```diff
@@ -1617,6 +1689,12 @@ msgstr "Przeład&uj"
 msgid "Ready!"
 msgstr "Gotowy!"
 
+msgid "Reason"
+msgstr "Przyczyna"
+
+msgid "Record stereo downmix"
+msgstr "Nagrywać dźwięk wielokanałowy w stereo"
```
Comment (pawel-szajna) [inline po/pl.po:1696]:
```suggestion
msgstr "Nagrywaj dźwięk wielokanałowy w stereo"
```

File: po/pl.po [hunk 10]
```diff
@@ -1676,6 +1757,9 @@ msgstr "Ładowanie sampli"
 msgid "Sample Size:"
 msgstr "Rozmiar sampli:"
 
+msgid "Sample Rate:"
+msgstr "Częstotliwość sampli:"
```
Comment (pawel-szajna) [inline po/pl.po:1761]:
```suggestion
msgstr "Częstotliwość próbkowania:"
```

File: po/pl.po [hunk 11]
```diff
@@ -1736,21 +1823,39 @@ msgstr "Wybierz plik pulsu"
 msgid "Select directory for cache store"
 msgstr "Wybierz folder do przechowywania pamięci podręczne"
 
+msgid "Select directory for setting import/export"
+msgstr "Wybierz folder do eksporta/importa ustawień"
```
Comment (pawel-szajna) [inline po/pl.po:1827]:
```suggestion
msgstr "Wybierz folder do importu/eksportu ustawień"
```

File: po/pl.po [hunk 12]
```diff
@@ -2062,6 +2179,9 @@ msgstr "na licencji GNU GPLv2 (lub nowszej)"
 msgid "menu"
 msgstr "meni"
 
+msgid "Metronome"
+msgstr "Taktomierz"
```
Comment (pawel-szajna) [inline po/pl.po:2183]:
```suggestion
msgstr "Metronom"
```

I've never ever heard any musician use the name "taktomierz", even though it apparently is a valid one. Still a valuable thing to learn :D

**Review by pawel-szajna — CHANGES_REQUESTED** [review]

File: po/pl.po (hunk 1 above)
Comment (oleg68) [reply po/pl.po:578]: I think all settings should keep a single style: either imperatives or the nouns. Please choose it and suggest your variants with the
```
+
-
```
icon

File: po/pl.po (hunk 4 above)
Comment (pawel-szajna) [reply po/pl.po:1301]: I think I've got them all!

File: po/pl.po (hunk 1 above)
Comment (pawel-szajna) [reply po/pl.po:578]: Well, let me take a second then and I'll go through the entire file to align them.

File: po/pl.po [hunk 13]
```diff
@@ -1736,21 +1823,39 @@ msgstr "Wybierz plik pulsu"
...
+msgstr "Wybierz folder do pakietów organowych"
+
+msgid "Select directory for your samplesets"
+msgstr "Wybierz folder do zestawów próbek"
+
 msgid "Send"
 msgstr "Wyślij"
 
+msgid "Send MIDI Recorder Output Stream to:"
+msgstr "Wyślij strumień wyjściowy rejestratora MIDI do:"
```
Comment (pawel-szajna) [inline po/pl.po:1851]:
```suggestion
msgstr "Wysyłanie strumienia wyjściowego rejestratora MIDI do:"
```

**Comment by pawel-szajna** [comment]
Well, the changed stuff seems to be pretty much done now, however the Polish translation might need further refinement as there are some unwritten rules followed by most of the Polish UIs, for instance with the imperative/noun forms I've mentioned before, buttons with actions tied to them usually use the imperative form and checkboxes use the noun form. I can't really do this tonight, but I'll try to go through the entire Polish translation on the weekend checking the translations with the actual UI.

File: po/pl.po (hunk 10 above)
Comment (oleg68) [reply po/pl.po:1761]: Now sometimes 'sampli', sometimes 'probki' are used in the ui. May be we choose one of them?

Comment (pawel-szajna) [reply po/pl.po:1761]: That's due to another language curiosity: when talking about samples as in sample sets, the imported word (sample, sampel, sampli, etc.) is often used. When signal processing is discussed (what I believe is what this "sample rate" describes), the term "próbkowanie" is used almost exclusively. While "próbka" and its variants could be used (and be understood) in the other contexts (próbki organów, sample organów both seem OK), when talking about signal processing "częstotliwość sampli" sounds unnatural.

Comment (oleg68) [reply po/pl.po:1761]: But "Sample rate" and "Samples per buffer" are neighbours at the "Audio" page. Using different words for them would look strange.

Comment (pawel-szajna) [reply po/pl.po:1761]: Indeed, those all seem to be related to the signal processing part and probably should be changed as well.

Furthermore, from what I can see, "próbki" is also used for sample sets in many places. Probably the best way would be to use "próbki" universally and just drop the loanword which may sound strange in some contexts anyway.

Comment (oleg68) [reply po/pl.po]: OK. I've replaced both with ``Częstotliwość próbkowania`` and ``Próbek na bofor``

**Review by pawel-szajna — APPROVED** [review]

**Comment by oleg68** [comment]
> Well, the changed stuff seems to be pretty much done now,

Thank you for your collaboration.

> buttons with actions tied to them usually use the imperative form and checkboxes use the noun form. 

Does it depend on the time when the action is taken?

For example, the button `Matching...` is now translated as `Dopasowanie...`, not `Dopasowaj...`, because when the user clicks this button, GrandOrgue does not match anything. Instead it shows the dialog for setting the rules for matching that will be done later.

> I can't really do this tonight, but I'll try to go through the entire Polish translation on the weekend checking the translations with the actual UI.

OK. You can take [an intermediate build with the recent po.pl](https://github.com/oleg68/GrandOrgue-official/actions/runs/1763033124) for futher refining.

Could you approve this pull request now?

**Comment by oleg68** [comment]
@larspalo @rousseldenis could you approve this PR?

**Comment by pawel-szajna** [comment]
>For example, the button Matching... is now translated as Dopasowanie..., not Dopasowaj..., because when the user clicks this button, GrandOrgue does not match anything. Instead it shows the dialog for setting the rules for matching that will be done later.

I believe that's how it works. I've just looked around some software and it seems that when immediate action is tied with some button, it's going to have the imperative form on it. It's curious how some things sound natural, some seem out of place, but I have to stop for a moment and think about it before I can really tell why. I wonder if anyone did some scientific research on how language is reflected in software.

>Thank you for your collaboration.

No problem at all, I think it's a nice thing to be able contribute a little bit to the software I've been using for several years now ☺️

**Review by larspalo — APPROVED** [review]

**Comment by oleg68** [comment]
@rousseldenis S'il vous plait, pouriez-vous approuver ce PR? Merci.

**Review by rousseldenis — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**